assets/css/argon.css

```diff
@@ -14387,6 +14387,7 @@ button.section-darker:focus
     width: 2.375rem;
     height: 2.375rem;
     padding: 0;
+    margin: 0 0 0.5rem 0;
```

Comment:
You can use bootstrap classes instead of using custom CSS as @YoshithaRathnayake said. Just try adding the `mb-2` class to all the buttons.

Reply:
So @Piumal1999 we can use mt-2 also right?

Reply:
Margin bottom or top both are working right?

Reply:
> So @Piumal1999 we can use mt-2 also right?

It will add an extra margin on the top of the first row of icons. Anyway, both are fine.